library/src/main/java/io/andref/rx/widgets/ListViewCard.java

```diff
@@ -183,8 +182,18 @@ private void layoutViews()
                     Log.e(TAG, "Drawable resource not found: " + exception.getMessage());
                 }
 
-                itemObservables.add(
-                        RxView.clicks(view)
+                RxView.clicks(view)
+                    .map(new Func1<Void, Item>()
+                    {
+                        @Override
+                        public Item call(Void aVoid)
+                        {
+                            return item;
+                        }
+                    })
+                    .subscribe(mItemClicks);
```

Comment:
If we want to not have subscriptions for every view, we will need to switch to using the adapter pattern for this view. Any work we do to support managing multiple views within this one would just be duplicating that pattern.